ui/app/styles/pages/auth-page.scss

```diff
@@ -31,8 +31,10 @@
 
     .icon.brand {
       display: flex;
+      width: scale.$lg-14;
+      height: scale.$lg-14;
 
-      path {
+      use {
```

Comment:
this is not how we would expect color to be applied. you're manipulating directly the internal SVG but we expose a `@color` prop for this (see https://flight-hashicorp.vercel.app/engineering#example-fill)

plus, our icons use `currentColor` by default so they can just inherit the color from the parent container

plus, our icons are flat, they don't contain strokes, so the `stroke: $brand-color;` declaration can be removed